.github/workflows/snapshotter-test.yml

```diff
@@ -0,0 +1,32 @@
+name: snapshotter-test
```

Comment:
Since it's an analogue with `test` but using the snapshotter, should it be `test-snapshotter`? Then it should sort adjacent-to but below the `test` workflow, I hope.

This occurred to me when I was thinking about whether we'd want to do split out the snapshotter test job for `windows-2022` the same way, and if so, what to call it.

Also, thinking about future naming if and when the snapshotter becomes the default, i.e. in `test`, what will the non-snapshotter test be named? `non-snapshotter-test` is very English-like, but sorts weirdly, and the most important aspect (`test`) is way over at the end.

Reply:
We could call them `test-graphdriver` and `test-snapshotter`

Reply:
> We could call them `test-graphdriver` and `test-snapshotter`

Works for me 👍 

(GitHub grouping / listing checks remains horrible in either case 😂)

Reply:
Guess it's too much to ask to have this sorter alphabetically 
<img width="312" alt="Screenshot 2023-09-29 at 18 13 38" src="https://github.com/moby/moby/assets/99933/6d32a938-6a7b-4a4a-9aae-e0d3e0cfe23b">